libs/shared/error/src/lib/sanitizeExceptionsDecorator.ts

```diff
@@ -3,9 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as Sentry from '@sentry/nextjs';
-import { Inject } from '@nestjs/common';
-import { ILogger } from '@fxa/shared/log';
-import { LOGGER_PROVIDER } from '@fxa/shared/log';
+import { Inject, Logger } from '@nestjs/common';
```

Comment:
Could the logger statement used in this file be updated to follow the same format as other log statements?

Reply:
This class is not managed by the nest app, and so needs to be directly injected. Was that what you were asking about?

Reply:
Oh no, sorry, should have given an example, I meant the actual log statements. I believe the aim was to have the first argument follow a dot notation? e.g. `logger.error('SanitizeExceptions.handleException', ....)`

```
  logger.error(
    `Error caught by SanitizeExceptions decorator in ${className}.${methodName}`,
    error
  );
```

Reply:
oh, good catch! yes, it absolutely should